test/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ foreach (bustub_test_source ${BUSTUB_TEST_SOURCES})
             --gtest_color=yes
             --gtest_output=xml:${CMAKE_BINARY_DIR}/test/${bustub_test_name}.xml
             --gtest_catch_exceptions=0
+            PROPERTIES
+            TIMEOUT 30
```

Comment:
Is this what is now `DISCOVERY_TIMEOUT` or is this test timeout? If the former, OK (but arguably we should use the new form instead, as of `3.10.3` it got renamed to avoid this confusion). If the latter, how/where will we handle tests > 30s?

Reply:
This is test timeout. Test cases > 30s will be considered failure if invoked using `ctest`. If invoked directly by the binary (e.g., ./test/binder_test), then it doesn't take effect.